doc/source/how-to/documenting.rst

```diff
@@ -0,0 +1,587 @@
+Documenting
+===========
+PyAnsys documentation must not only be written but also maintained. If you are
+new to writing developer documentation, see the `Google Developer Documentation
+Style Guide <https://developers.google.com/style>`_. It provides
+editorial guidelines for writing clear and consistent developer documentation,
+allowing this guide to supply guidance specific to PyAnsys library
+documentation.
+
+When writing developer documentation, the relationship between code and
+documentation is key. To keep documentation up to date with radial evolving
+code:
+
+- Minimize the content footprint
+- Write `timeless documentation <https://developers.google.com/style/timeless-documentation>`_
+- Support contributions from both inside and outside of the development team
+- Perform periodic reviews
+
+Documentation Sources
+---------------------
+.. raw:: html
+    
+    <div align="center">
+      <img src="https://github.com/sphinx-doc/sphinx/raw/5.x/doc/_static/sphinx.png">
+    </div>
+    <br>
+
+The generation of PyAnsys documentation uses `Sphinx
+<https://www.sphinx-doc.org/en/master/>`__ and an Ansys-branded theme
+(`ansys-sphinx-theme <https://github.com/ansys/Sansys-sphinx-theme>`_) to
+assemble content in:
+
+- Docstrings
+- reStructuredText (RST) files
+- Python (PY) example files
+
+Docstrings
+~~~~~~~~~~
+Docstrings must be formatted so that Sphinx can parse them. You can use either
+of these Sphinx extensions:
+ 
+- `numpydoc <https://pypi.org/project/numpydoc/>`_
+- `napoleon <https://pypi.org/project/sphinxcontrib-napoleon/>`_
+
+The ``numpydoc`` extension is generally preferred because it supports an API
+documentation structure with one page per method, providing Python community
+members with documentation like that generated by `pandas <https://pandas.pydata.org/>`_
+and `NumPy <https://numpy.org/>`_. If, however, your API is very linear, you
+might prefer the  ``napoleon`` extension because it supports a documentation
+structure where everything needed to solve a certain problem can be shown on one page.
+
+The ``numpydoc`` extension provides its own `style guide
+<https://numpydoc.readthedocs.io/en/latest/format.html>`_ and a `user guide
+<https://numpydoc.readthedocs.io/en/latest/>`_ that explains how to use the
+extension with Sphinx. The ``napoleon`` extension, which parses both numpy and
+Google style docstrings, refers you to the `Google Python Style Guide
+<https://google.github.io/styleguide/pyguide.html>`_.
+
+Regardless of the extension that you choose for generating documentation, we
+recommend the use of numpy-style docstrings so that there is consistency
+within PyAnsys libraries. For more information, see :ref:`Documentation Style`.
+
+RST Files
```

Comment:
We need to check if this is the first reference to RST files--it needs to defined as ReStrucutred Text (RST) files on first reference before RST can be used subsequently.